src/figuresClasses.ts

```diff
@@ -2,10 +1,74 @@
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('A sides must be greater than zero.');
+    }
```

Comment:
[CODE STYLE] If you throw an error try to provide its detailed description in error message. For example, instead of 'A sides must be greater than zero.', you could write 'All sides of a triangle must be greater than zero.' to make it more clear.